recaf-assembler/src/main/java/me/coley/recaf/assemble/transformer/AstToMethodTransformer.java

```diff
@@ -35,13 +35,12 @@ public class AstToMethodTransformer {
 	private ExpressionToAstTransformer exprToAst;
 	private ClassSupplier classSupplier;
 	private String selfType;
-	private Unit unit;
 	private MethodDefinition definition;
 	private Code code;
 	// Configurable
 	private InheritanceChecker inheritanceChecker = ReflectiveInheritanceChecker.getInstance();
 	private boolean doLimitVarRange = true;
-	private boolean useAnalysis = true;
+	private boolean useAnalysis = false;
```

Comment:
Should be `true`?

Reply:
Its set in the `AnalysisPipeline`, if anything the assignment can be removed for brevity.